pkg/cwhub/helpers.go

```diff
@@ -0,0 +1,146 @@
+package cwhub
+
+import (
+	"fmt"
+
+	"github.com/crowdsecurity/crowdsec/pkg/csconfig"
+	"github.com/enescakir/emoji"
+	log "github.com/sirupsen/logrus"
+)
+
+func InstallItem(csConfig *csconfig.Config, name string, obtype string, force bool, downloadOnly bool) error {
+	it := GetItem(obtype, name)
+	if it == nil {
+		return fmt.Errorf("unable to retrieve item : %s", name)
+	}
+	item := *it
+	if downloadOnly && item.Downloaded && item.UpToDate {
+		log.Warningf("%s is already downloaded and up-to-date", item.Name)
+		if !force {
+			return nil
+		}
+	}
+	item, err := DownloadLatest(csConfig.Hub, item, force, false)
```

Comment:
```suggestion
item, err := DownloadLatest(csConfig.Hub, item, force, true)
```
Doing so will fix the test `TestUpgradeConfigNewScenarioInCollection`